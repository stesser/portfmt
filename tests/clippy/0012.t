USES=		shebangfix
sh_OLD_CMD=	foo
sh_CMD=		foo

.include <bsd.port.mk>
<<<<<<<<<
# USES block
USES

# Unknown variables
# WARNING:
# The following variables were not recognized.
# They could just be typos or Portclippy needs to be made aware of them.
# Please double check them.
#
# Prefix them with an _ or wrap in '.ifnmake portclippy' to tell
# Portclippy to ignore them.
#
# If in doubt please report this on portfmt's bug tracker:
# https://github.com/t6/portfmt/issues
+sh_CMD
sh_OLD_CMD
-sh_CMD
