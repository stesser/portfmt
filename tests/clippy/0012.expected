# USES block
USES

# Unknown variables
# WARNING:
# The following variables were not recognized.
# They could just be typos or Portclippy needs to be made aware of them.
# Please double check them.
#
# Prefix them with an _ to tell Portclippy to ignore them.
#
# If in doubt please report this on portfmt's bug tracker:
# https://github.com/t6/portfmt/issues
+sh_CMD
sh_OLD_CMD
-sh_CMD
